src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -505,27 +518,23 @@ public Game getByUid(int uid) {
     return game;
   }
 
-  public void addOnRankedMatchNotificationListener(Consumer<MatchmakerInfoMessage> listener) {
-    fafService.addOnMessageListener(MatchmakerInfoMessage.class, listener);
-  }
-
-  public CompletableFuture<Void> startSearchLadder1v1(Faction faction) {
+  public CompletableFuture<Void> startSearchMatchmaker() {
     if (isRunning()) {
-      log.debug("Game is running, ignoring 1v1 search request");
+      log.debug("Game is running, ignoring matchmaking search request");
       notificationService.addImmediateErrorNotification(new UnsupportedOperationException("Forged Alliances is already running"), "game.gameRunning");
       return completedFuture(null);
     }
 
     if (!preferencesService.isGamePathValid()) {
       CompletableFuture<Path> gameDirectoryFuture = postGameDirectoryChooseEvent();
-      return gameDirectoryFuture.thenCompose(path -> startSearchLadder1v1(faction));
+      return gameDirectoryFuture.thenCompose(path -> startSearchMatchmaker());
     }
 
-    searching1v1.set(true);
+    inMatchmakerQueue.set(true);
 
-    return modService.getFeaturedMod(LADDER_1V1.getTechnicalName())
+    return modService.getFeaturedMod(FAF.getTechnicalName())//TODO: use matchmaking featured mod
```

Comment:
Can this TODO be resolved?

Reply:
FAF is the right featured mod for the matchmaker. I will delete the todo